src/main/java/com/microsoft/spring/data/gremlin/conversion/GremlinScriptEdgeFindLiteral.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.conversion;
+
+import com.microsoft.spring.data.gremlin.common.Constants;
+import com.microsoft.spring.data.gremlin.exception.UnexpectedGremlinSourceTypeException;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+import org.springframework.util.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+public class GremlinScriptEdgeFindLiteral implements GremlinScript<String> {
```

Comment:
Add some comments for this class？

Reply:
Why do we separate edge operations and vertex operations into different classes? Is there any combined operations?

Reply:
As you cannot query both edge and vertex at the same time. They all implement Script and generate the script for query. If put all together, we cannot restrict them by interface.